nav2_simple_commander/nav2_simple_commander/robot_navigator.py

```diff
@@ -274,17 +278,17 @@ def waitUntilNav2Active(self, navigator='bt_navigator', localizer='amcl'):
         self.info('Nav2 is ready for use!')
         return
 
-    def getPath(self, start, goal, planner_id=None):
```

Comment:
We could also use the "None way" everywhere (set the variable if it is not None)